### ISSUE TITLE
Add `--color=auto/always/never`

### ISSUE THREAD

**Issue by cblp** [ff-notes/ff#137]

**Comment by willbasky** [comment]
What `auto` acts?

**Comment by cblp** [comment]
added to the description

**Comment by willbasky** [comment]
> auto [default] — detect if we are running in a terminal (isatty?) and colorize only if in terminal

Can `ff` be launched not in terminal?

**Comment by cblp** [comment]
Yes, when launched from a script or a command, like `ff | less`, stdout is a pipe, not a terminal (teletype, tty).
